tests/more_string_manipulation/test_challenge_080.py

```diff
@@ -0,0 +1,22 @@
+# MORE STRING MANIPULATION
+from exercises.more_string_manipulation import challenge_080
+
+
+def test_processing():
+    # Define inputs
+    test_first_name = "Marco"
+    test_last_name = "Barr"
+
+    # invoke function with the test inputs
+    (
+        length_user_first_name,
+        length_user_last_name,
+        length_full_name,
+        full_name,
+    ) = challenge_080.processing(test_first_name, test_last_name)
+
+    # verify that function output is as expected
+    assert length_user_first_name == 5
```

Comment:
```suggestion
    assert length_user_first_name == len(test_first_name)
```
And this change should be repeated elsewhere so the expectations are ideally expressions of the input values to the test